src/lib/rpcWallet.js

```diff
@@ -307,10 +307,12 @@ export default {
   async [AEX2_METHODS.LOGOUT]() {
     const { clients: aepps } = this.sdk.getClients();
     Array.from(aepps.values()).forEach(aepp => {
-      aepp.sendMessage(
-        { method: 'connection.close', params: { reason: 'bye' }, jsonrpc: '2.0' },
-        true,
-      );
+      if (aepp.info.status !== 'DISCONNECTED') {
```

Comment:
In this case, we are sending `connection.close` if the status **is not** `DISCONNECTED`. Is that the desired logic?

Reply:
yes